content/en/blog/2019/v1beta1-authorization-policy/index.md

```diff
@@ -0,0 +1,283 @@
+---
+title: Introducing the Istio v1beta1 Authorization Policy
+description: Introduction, motivation and design principles for the Istio v1beta1 Authorization Policy.
+publishdate: 2019-11-11
+subtitle:
+attribution: Yangmin Zhu (Google)
+keywords: [security, RBAC, access control, authorization]
+target_release: 1.4
+---
+
+Up until now, Istio has provided the role based access control (RBAC) policy for enforcing access control
+on services using three configuration resources: `ClusterRbacConfig`, `ServiceRole` and `ServiceRoleBinding`.
+With this API, users have been able to enforce access control in mesh-level, namespace-level and
+service-level. Like any other RBAC policy, Istio RBAC uses the same concept of role and binding for granting
+permissions to identities.
+
+While this functionality has proven to be working for many customers, we have also recognised
+some shortcomings in its design that is causing many confusions and preventing its further adoption.
+
+For example, some users have had the wrong impression that the enforcement happens in service-level because
+`ServiceRole` uses service to specify where to apply the policy, however, the policy is actually applied
+in workload-level, the service is only used to find the corresponding workload. This has some big impact, especially
+when multiple services are referring to the same workload. A `ServiceRole` for service A will also affect
+service B if the two services are referring to the same workload, this surprises customers and could even cause
+security vulnerability.
+
+The other example is it's a little bit hard to maintain and manage the Istio RBAC configurations because three
+are three different customer resource definitions in Istio RBAC. Istio has already had an issue that there
+are too much CRDs to learn and manage, and the learning and maintenance burden of having three different configurations
+in Istio RBAC turned out to be greater than its benefit.
+
+To address these, and other issues, Istio 1.4 introduces the `v1beta1` authorization policy which will
+completely replace the previous `v1alpha1` RBAC policy going forward. The `v1beta1` policy provides
+more features but is not backward compatible and will require manual conversion from the old API (we
+will provide tool to help automate the conversion).
```

Comment:
link to tool/docs to migrate?

Reply:
The doc for the tool is still WIP, will add the link when it's merged. @pitlv2109